docs/ai-coder/ai-bridge/monitoring.md

```diff
@@ -60,3 +60,68 @@ needs.
 
 For configuration options and details, see [Data Retention](./setup.md#data-retention)
 in the AI Bridge setup guide.
+
+## Tracing
+
+AI Bridge supports tracing via [OpenTelemetry](https://opentelemetry.io/),
+providing visibility into request processing, upstream API calls, and MCP server
+interactions.
+
+### Enabling Tracing
+
+AI Bridge tracing is enabled when tracing is enabled for the Coder server.
+To enable tracing set `CODER_TRACE_ENABLE` environment variable or
+[--trace](https://coder.com/docs/reference/cli/server#--trace) CLI flag:
+
+```sh
+export CODER_TRACE_ENABLE=true
+```
+
+```sh
+coder server --trace
+```
+
+Open Telemetry configuration must be provided by environment variables.
+See: https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/protocol/exporter.md.
```

Comment:
Let's rather not duplicate this. Once we have a dedicated page we can link to it.

```suggestion
```